melt_czochralski_Dadzis/base_classes.schema.archive.yaml

```diff
@@ -40,81 +29,14 @@ definitions:
                   eln:
                     component: ReferenceEditQuantity  
     Experiment: 
-      base_sections: 
-      - nomad.datamodel.metainfo.eln.ElnBaseSection
-      - '#/Activity'
+      base_section:  '#/Entity'
```

Comment:
Why Experiment is now Entity? It should be either Activity or Workflow

Reply:
This is the file currently used in multilog, I just copied it from here. If the present version in this repo is better I'll replace it!

Reply:
Yes, I didn't mention that the most updated is in the base_classes folder, so could you replace this with the new one please? You should also test if the upload is still successful with the new base_classes.schema.archive.yaml